app/views/organization_timeline_entries/index.html.erb

```diff
@@ -19,12 +22,25 @@
     <% @entries.each do |entry| %>
       <% if can?(:read, entry) %>
         <tr>
-          <td class="hidden-xs"><%= link_to date_and_time(entry.started_at), edit_organization_timeline_entry_path(entry) %></td>
+          <td class="hidden-xs"><%= link_to date_and_time(entry.started_at), organization_timeline_entry_path(entry) %></td>
           <td class="hidden-xs"><%= date_and_time entry.ended_at unless entry.ended_at.blank? %></td>
```

Comment:
Don't use this path, it breaks things.  Use downtime_path instead.  This model definitely could be designed better but for now lets just do this.